### PR TITLE
Upgrade to dns-packet 4.0.0

### PR DESCRIPTION
**Pull request by richardschneider**
Some minor changes to support dns-packaet 4.0.0

- `class` is now a string (not a number)
- `flag` names have changed
- `TXT data` is an array of buffers

### CODE REVIEW

**Comment by richardschneider** [comment]
@mafintosh Any comments?

**Comment by mafintosh** [comment]
No this is great. So this is a major bump in this package also right?

File: test.js [hunk 1]
```diff
@@ -160,17 +160,17 @@ tests.forEach(function (test) {
   })
 
   test('TXT record', function (dns, t) {
-    var data = Buffer.from('black box')
+    var data = [Buffer.from('black box')]
```
Comment (mafintosh) [inline test.js]: Mind adding a test here where the array contains more than one item also?

**Comment by mafintosh** [comment]
@richardschneider added a minor comment :)

File: test.js (hunk 1 above)
Comment (richardschneider) [reply test.js]: No worries.

**Comment by richardschneider** [comment]
Yes, I believe this is a major bump because `dns-packet` has changed it's interface.

**Comment by richardschneider** [comment]
@mafintosh RTM?

**Comment by mafintosh** [comment]
7.0.0, thanks again @richardschneider 🎉

**Pull request merged**